### PR TITLE
HARVESTER: Mutil-cluster import harvester enhancement

### PR DESCRIPTION
**Pull request by n313893254**
- https://github.com/rancher/dashboard/issues/4531
- https://github.com/harvester/harvester/issues/1030

**Additional Context**

[Import cluster instuction]
image

[Setting Config]
image

[Machine Config]
image

### CODE REVIEW

**Review by nwmac — APPROVED** [review]
Approved - I am only checking the changes to the core Rancher code to make sure there is nothing that might affect Rancher.

**Comment by n313893254** [comment]
@richard-cox Solved the merge conflicts.

We can import more than one harvester instance for a rancher install.  I don't think the `Import Existing` should be removed.

**Comment by richard-cox** [comment]
@n313893254  Ah, thought I saw somewhere in one of the linked issues it was 1:1. To confirm then the flow for importing multiple harvester clusters is to change the Global Setting --> Cluster Registration URL every time? If so I think given the time we should still merge this PR, but look to improve this later. I'll bring up with @nwmac

**Comment by guangbochen** [comment]
Yep, that is 1:1 mapping of the Harvester cluster and the upstream Rancher, also the cluster importing will be managed on the Harvester side via the `cluster-registration-url` setting config. So on the Rancher side, we are changing the UI display only.

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -1073,6 +1073,27 @@ cluster:
       =1 {is 1 Harvester cluster that is not shown}
       other {are # Harvester clusters that are not shown}
       }  
+    registration:
+      step1: "1. Go to the settings page of Harvester UI."
```
Comment (richard-cox) [inline assets/translations/en-us.yaml:1077]: Now I understand the issue, I think this should be updated to 

`1. Go to the settings page of the target Harvester's UI`

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: assets/translations/en-us.yaml (hunk 1 above)
Comment (n313893254) [reply assets/translations/en-us.yaml]: Done

**Review by richard-cox — APPROVED** [review]
@n313893254 I fixed a `&#39;` issue and also tweaked the suggested text. If you're happy with the below I'll merge

image

**Comment by guangbochen** [comment]
LGTM, @richard-cox please help to merge with your latest changes, thanks.

**Pull request merged**